.github/workflows/tests.yml

```diff
@@ -206,3 +206,31 @@ jobs:
         v=${GITHUB_REF:10}
         docker tag brew "docker.pkg.github.com/homebrew/brew/brew:$v"
         docker push "docker.pkg.github.com/homebrew/brew/brew:$v"
+
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tag: [ubuntu18, ubuntu20]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+      with:
+        fetch-depth: 0
+    - name: Build Docker image
+      run: docker build -t brew -f docker/${{matrix.tag}}/Dockerfile .
+    - name: Run brew test-bot
+      run: docker run --rm brew brew test-bot
```

Comment:
Do we need to run this for both OSs on every commit? It seems pretty heavy.

Reply:
We definitely need to run some command to test that the image works. This run of `brew test-bot` takes only nine seconds. Evidently it's only running `brew config`.

Reply:
I guess my thinking is: do we need to build three Docker containers for every commit? Couldn't we do something like only provide the default container for every commit and the others for stable tags? We don't really want non-maintainers using the non-tagged ones anyway?

Reply:
I've changed it so that 16.04 deploys on changes to `master`, and all three versions deploy on a tag.

Reply:
Perfect, thanks ❤️